### PR TITLE
nothing here make senses

### PR DESCRIPTION
**Pull request by llllOllOOll**
This is a crazy update. I made many changes in the source code. I add new features.
Don't hate for that!! lol

### CODE REVIEW

File: example/ameiva/falling_stars/helpers.dart [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+class Point {
+  double x, y;
+  Point(this.x, this.y);
+
+  // Operator Overlay
+  // Allow us change an operato like ===, or, + , -
+  // Here we change the minus to allow it adde a Vector to the current point
+  // The x and y the will be adde to x and y to the point and will be returned a new point.
```
Comment (ToniCalfim) [inline example/ameiva/falling_stars/helpers.dart:20]: // Operator Overlay
  //
  // Allows us to change an operator, just like ===, or, + , -
  //
  // Here we change the minus signal to allow it to add a Vector to the current point.
  // The x and y will be added to the x and y point and will returna new point.

File: example/ameiva/falling_stars/helpers.dart [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  Point operator -(Vector vector) {
```
Comment (ToniCalfim) [inline example/ameiva/falling_stars/helpers.dart:21]: Point operator - (Vector vector) {

File: example/ameiva/falling_stars/helpers.dart [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    Point potision = Point(0, 0);
```
Comment (ToniCalfim) [inline example/ameiva/falling_stars/helpers.dart:22]: Please, change "**potision**" to "_position_" in the whole document...

**Review by ToniCalfim — COMMENTED** [review]
Getting used to reviews...

File: example/ameiva/falling_stars/helpers.dart (hunk 3 above)
Comment (llllOllOOll) [reply example/ameiva/falling_stars/helpers.dart:22]: I will make all that changes tomorrow.

**Pull request merged**